app/utils/regexp.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-cli-app-version/utils/regexp';
```

Comment:
Is it necessary to re-export these utilities in the `app` tree? I think that the `util` generator probably just created this file by default, but keeping this file here would define

```js
Import regexp from 'consuming-app/utils/regexp';
```

Which is probably not desired.